app/main/views/g_cloud.py

```diff
@@ -33,8 +33,11 @@
 from app import search_api_client, data_api_client, content_loader
 
 
+PROJECT_END_SEARCH_LIMIT = 100
 PROJECT_SAVED_MESSAGE = Markup("""Search saved.""")
-PROJECT_ENDED_MESSAGE = Markup("""Your search has been ended. You can now download your shortlist.""")
+PROJECT_ENDED_MESSAGE = Markup("""Search ended. You can now download your shortlist.""")
+PROJECT_TOO_MANY_RESULTS_ERROR = Markup("""
+    You have too many results. Choose a category or add filters to refine your search.""")
```

Comment:
FWIW I don't think we necessarily have to use the `Markup` thing if we aren't actually including any markup :)

Reply:
Oh, the PROJECT_ thing here wasn't supposed to be a prefix. It makes sense for 'project ended' because that's what ended - these should just be END_SEARCH_LIMIT and TOO_MANY_RESULTS_ERROR